docs/rbac.rst

```diff
@@ -0,0 +1,269 @@
+Role Based Access Control
+=========================
+
+.. important::
+
+    This feature is a Tech Preview and can be subject of change.
+
+.. important::
+
+    Using user ``admin`` will bypass all RBAC checks.
+
+
+* `Permission` : Permission to do **one** action on one `viewset` as a `view` or a `create`.
```

Comment:
Should "as a" be ", for example" or "e.g."? I'm not sure what the intention is here

Reply:
This section should explain basic terms.

Reply:
But I would remove it.